recipes/daggy/2.1/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.21.0)
+project(DaggyPackageTest)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(daggy REQUIRED)
+
+add_subdirectory(pingpong)
```

Comment:
We like to the test packages small perhaps we can remove this one since it's not being run (and it's coverd by the other two)

```suggestion
```

Reply:
To say you the truth, this project is needed for testc and testcpp - both projects are run this process. The Daggy lib is about processes control, therefore I did simple process project.
From other side, I understand you afraid about testing while package building - now my test_package doing small test. I will think about.